client/deployment/src/main/resources/templates/libraries/microprofile/pojo.qute

```diff
@@ -1,5 +1,6 @@
 {@org.openapitools.codegen.CodegenModel m}
 import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
```

Comment:
Instead of importing, can you please use the FQCN to avoid compilation errors due to possible ambiguities?

Reply:
> Instead of importing, can you please use the FQCN to avoid compilation errors due to possible ambiguities?

Ofc, I will change so it uses FQCN instead. I tried to match the existing patterns and StringBuilder doesn't have use FQCN.

Reply:
Just to complement, we still have some imports, like `com.fasterxml.jackson.annotation.JsonProperty` in the above line. But the idea is to remove them and not add more.